TravelTimeAgent/Jobson.cs

```diff
@@ -146,26 +146,17 @@ public bool Execute(Double? InitialMass_M_i_kg=null, DateTime? starttime = null)
 
                 double distance = 0.0;
 
-                for (int i = 1; i < this.Reaches.Count; i++)
+                for (int i = 1; i <= this.Reaches.Count; i++)
```

Comment:
If you change this to `for (int i = 0; i < this.Reaches.Count; i++)`, you can change all of the places where you use `i - 1` to just `i`

Reply:
Because this is a temporary fix, I would prefer to simply add the = rather than make more changes in the code. I will, however, add documentation as to what needs to be reverted back after this temporary period.